src/backend/joanie/core/migrations/0049_alter_contractdefinition_name.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 4.2.16 on 2024-12-02 17:20
+
+from django.db import migrations, models
+
+
+def migrate_contract_definition_name(apps, schema_editor):
+    """
+    Updates all existing ContractDefinition records with the name 'contract_definition'
+    to 'contract_definition_unicamp'. This ensures consistency with the new naming convention.
+    """
+    ContractDefinition = apps.get_model("core", "ContractDefinition")
+    ContractDefinition.objects.filter(
+        name="contract_definition"
+    ).update(name="contract_definition_unicamp")
+
+
+def rollback_contract_definition_name(apps, schema_editor):
+    """
+    If we need to rollback, this method should be called to reverse the `contract_definition`
+    name to the original ones.
+    """
+    ContractDefinition = apps.get_model("core", "ContractDefinition")
+    for contract_definition in ContractDefinition.objects.iterator():
+        contract_definition.name = "contract_definition"
+        contract_definition.save()
```

Comment:
Just for information, this is not mandatory to apply this suggestion but this should work:

```suggestion
ContractDefinition.objects.update(name="contract_definition")
```

Reply:
it saves some lines, I've applied them ! thanks for the tips